core/src/main/java/ar/com/wolox/wolmo/core/activity/WolmoActivityHandler.kt

```diff
@@ -19,49 +19,40 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package ar.com.wolox.wolmo.core.activity;
+package ar.com.wolox.wolmo.core.activity
 
-import android.os.Bundle;
+import android.os.Bundle
+import ar.com.wolox.wolmo.core.R
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import javax.inject.Inject
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import javax.inject.Inject;
-
-import ar.com.wolox.wolmo.core.R;
-import ar.com.wolox.wolmo.core.util.ToastFactory;
-
-public class WolmoActivityHandler {
-
-    private ToastFactory mToastFactory;
-    private WolmoActivity mWolmoActivity;
-
-    @Inject
-    WolmoActivityHandler(ToastFactory toastFactory) {
-        mToastFactory = toastFactory;
-    }
+class WolmoActivityHandler @Inject constructor(
+        private val mToastFactory: ToastFactory
```

Comment:
Remember the `m...` convention is just for Java, not for kotlin.

Reply:
I had not seen it, done!